mix_format_file.py

```diff
@@ -74,4 +66,30 @@ def run(self, edit):
       window.run_command('hide_panel', {'panel': 'output.mix_format'})
       window.status_message('Formatting successful!')
 
+  def find_cwd(self, window, settings, file_name):
+    cur_dir = path.dirname(file_name)
+    cwd = os.path.expanduser("~")
+    for root_path in window.folders():
```

Comment:
Curiosity: why do we need to use `window.folders()` instead `window.folders()[0]`?

Reply:
https://tppr.me/5B4EV
when projects with different .formatter.exs
with window.folders () [0] we can use the wrong config file